### PR TITLE
Switch remaining stragglers to new Guile mechanism

### PR DESCRIPTION
**Pull request by qookei**
Hopefully these should be the last packages that need new revisions.

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: dev-scheme/guile/guile-1.8.8-r100.ebuild [hunk 1]
```diff
@@ -128,10 +128,8 @@ src_install() {
 	mv "${ED}"/usr/share/aclocal/guile{,-"${SLOT}"}.m4 || die
 	find "${ED}" -name '*.la' -delete || die
 
-	# GUILE_LOAD_PATH for TeXmacs (bug #23493)
 	newenvd - "50guile${SLOT}" <<-EOF
 	PKG_CONFIG_PATH="${GUILE_PCDIR}"
-	GUILE_LOAD_PATH="${EPREFIX}/usr/share/guile/${SLOT}"
```
Comment (thesamesam) [inline dev-scheme/guile/guile-1.8.8-r100.ebuild:134]: Let's revbump it to be safe, however unlikely it is that anyone installed it.

File: app-emacs/geiser-guile/geiser-guile-0.28.3-r100.ebuild [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+PATCHES=( "${FILESDIR}/${PN}-guile-scheme-src-dir.patch" )
+
+DOCS=( readme.org )
+ELISP_TEXINFO="${PN}.texi"
+SITEFILE="50${PN}-gentoo.el"
+
+src_prepare() {
+	elisp_src_prepare
+
+	sed "s|@SITEETC@|${EPREFIX}${SITEETC}/${PN}|" -i "${PN}.el" || die
```
Comment (thesamesam) [inline app-emacs/geiser-guile/geiser-guile-0.28.3-r100.ebuild:47]: You can combine these into one `sed` with `sed -e`.

**Comment by ArsenArsen** [comment]
these are RT deps on guiles cli though, I think, so I'm not sure they should be handled quite like this?

**Comment by thesamesam** [comment]
We discussed it on IRC and there's not an alternative:
1) If something _depends_ on these packages, they need to know they have a working guile;
2) Nothing should rely on `/usr/bin/guile` existing anyway (nothing creates it right now, we don't even have `eselect-gulie`)

**Pull request closed without merging**